apps/web/pages/api/book/recurring-event.ts

```diff
@@ -53,16 +54,25 @@ async function handler(req: NextApiRequest & { userId?: number }, res: NextApiRe
       ...booking,
       appsStatus,
       allRecurringDates,
+      thirdPartyRecurringEventId,
       currentRecurringIndex: key,
       noEmail: key !== 0,
     };
 
     const eachRecurringBooking = await handleNewBooking(recurringEventReq, {
       isNotAnApiCall: true,
-      skipAvailabilityCheck: key >= numSlotsToCheckForAvailability,
+      skipAvailabilityCheck: key >= numSlotsToCheckForAvailability || thirdPartyRecurringEventId !== "",
```

Comment:
Don't we want to check for availability for the number of slots equal to `numSlotsToCheckForAvailability` now?

`thirdPartyRecurringEventId` could be non empty after the very first booking right? and then it would stop checking for availability

Reply:
Thanks for the catch, I now check if there are any conflicts in any of the instances in the first iteration of this loop.